### DISCUSSION TITLE
Começando com Angular Estilizando corpo da página

### DISCUSSION THREAD

**Discussion by erickcred** [balta-io/forum#350] [Tira Dúvidas] [answered]
Olá boa noite, tudo bom ? 

Estou realizando o curso de Angular (Começando com Angular) mas no vídeo (Estilizando corpo da página) o vídeo tem 9 min mas ele fica com tela preta aos 7:26 mim, está correto?

**Comment by andrebaltieri** [comment, marked as answer]
@erickcred obrigado pelo feedback, acredito que é um bug.
